Updates for new sensubility amqp1 address (#239)
infrawatch/documentation: doc-Service-Telemetry-Framework/modules/proc_planning-amqp-address-prefixes.adoc

= Planning AMQP address prefixes

[role="_abstract"]

ifdef::include_when_16[]

> By default, {OpenStack} nodes get data through three data collectors; collectd, Ceilometer. The collectd-sensubility plugin requires a unique address as if it was a separate collector. These components send telemetry data or notifications to the respective AMQP addresses, for example, `collectd/telemetry`, where {ProjectShort} Smart Gateways listen on those addresses for monitoring data. To support multiple clouds and to identify which cloud generated the monitoring data, configure each cloud to send data to a unique address. Prefix a cloud identifier to the second part of the address. The following list shows some example addresses and identifiers:

Reviewer: What's the third? :)

Author: I originally listed collectd-sensubility as one, looks like I forgot to change the number back!

> By default, {OpenStack} nodes get data through two data collectors; collectd and Ceilometer. The collectd-sensubility plugin requires a unique address as if it was a separate collector. These components send telemetry data or notifications to the respective AMQP addresses, for example, `collectd/telemetry`, where {ProjectShort} Smart Gateways listen on those addresses for monitoring data. To support multiple clouds and to identify which cloud generated the monitoring data, configure each cloud to send data to a unique address. Prefix a cloud identifier to the second part of the address. The following list shows some example addresses and identifiers:

Reviewer 2: ```suggestion
By default, {OpenStack} nodes receive data through two data collectors; collectd and Ceilometer. The collectd-sensubility plugin requires a unique address. These components send telemetry data or notifications to the respective AMQP addresses, for example, `collectd/telemetry`. {ProjectShort} Smart Gateways listen on those AMQP addresses for monitoring data. To support multiple clouds and to identify which cloud generated the monitoring data, configure each cloud to send data to a unique address. Add a cloud identifier prefix to the second part of the address. The following list shows some example addresses and identifiers:
```
Changed "get" to receive. Maybe the word "collect" is more appropriate?
Deleted this part because it will be difficult for translators and is it vital information anyway?: "as if it was a separate collector"

Reviewer: This is good with me. @pleimer ?

Author: works for me

* `collectd/cloud1-telemetry`
* `collectd/cloud1-notify`
* `sensubility/cloud1-telemetry`
* `anycast/ceilometer/cloud1-metering.sample`
* `anycast/ceilometer/cloud1-event.sample`
* `collectd/cloud2-telemetry`
* `collectd/cloud2-notify`
* `sensubility/cloud2-telemetry`
* `anycast/ceilometer/cloud2-metering.sample`
* `anycast/ceilometer/cloud2-event.sample`
* `collectd/us-east-1-telemetry`
* `collectd/us-west-3-telemetry`
endif::include_when_16[]

ifdef::include_when_13[]

> By default, OpenStack nodes get data through two data collectors; collectd and Ceilometer. These components send telemetry data or notifications to the respective AMQP addresses, for example, collectd/telemetry, where STF Smart Gateways listen on those addresses for monitoring data. To support multiple clouds and to identify which cloud generated the monitoring data, configure each cloud to send data to a unique address. Prefix a cloud identifier to the second part of the address. The following list shows some example addresses and identifiers:

Reviewer: ```suggestion
By default, OpenStack nodes get data through two data collectors; collectd and Ceilometer. These components send telemetry data or notifications to the respective AMQP addresses, for example, collectd/telemetry, where {ProjectShort} Smart Gateways listen on those addresses for monitoring data. To support multiple clouds and to identify which cloud generated the monitoring data, configure each cloud to send data to a unique address. Prefix a cloud identifier to the second part of the address. The following list shows some example addresses and identifiers:
```

> By default, OpenStack nodes get data through two data collectors; collectd and Ceilometer. These components send telemetry data or notifications to the respective AMQP addresses, for example, collectd/telemetry, where {ProjectShort} Smart Gateways listen on those addresses for monitoring data. To support multiple clouds and to identify which cloud generated the monitoring data, configure each cloud to send data to a unique address. Prefix a cloud identifier to the second part of the address. The following list shows some example addresses and identifiers:

Reviewer 2: ```suggestion
By default, {OpenStack} nodes get data through two data collectors; collectd and Ceilometer. These components send telemetry data or notifications to the respective AMQP addresses, for example, collectd/telemetry. {ProjectShort} Smart Gateways listen on the AMQP addresses for monitoring data. To support multiple clouds and to identify which cloud generated the monitoring data, configure each cloud to send data to a unique address. Add a cloud identifier prefix to the second part of the address. The following list shows some example addresses and identifiers:
```
